xclim/indicators/atmos/_temperature.py

```diff
@@ -394,10 +394,15 @@ class TempWithIndexing(ResamplingIndicatorWithIndexing):
     standard_name="cold_spell_duration_index",
     long_name="Number of days part of a percentile-defined cold spell",
     description="{freq} number of days with at least {window} consecutive days "
-    "where the daily minimum temperature is below the 10th "
-    "percentile. The 10th percentile should be computed for "
-    "a 5-day window centred on each calendar day in the  1961-1990 period",
+    "where the daily minimum temperature is below the {per_base_thresh}th "
+    "percentile(s). A {per_window} day(s) window, centred on each calendar day in the "
+    "{per_period} period, is used to compute the {per_base_thresh}th percentile(s).",
     cell_methods="",
+    default_params={
+        "per_base_thresh": 10,
+        "per_window": 5,
+        "per_period": "1961-1990",
+    },
```

Comment:
I understand what you are trying to do, but I think this assumes too much. I would prefer an error or an empty string, rather than false information.